prowler/providers/aws/services/guardduty/guardduty_service.py

```diff
@@ -69,6 +71,57 @@ def __get_detector__(self, regional_clients):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __get_administrator_account__(self, regional_clients):
+        logger.info("GuardDuty - getting administrator account...")
+        try:
+            for detector in self.detectors:
+                try:
+                    regional_client = regional_clients[detector.region]
+                    detector_administrator = regional_client.get_administrator_account(
+                        DetectorId=detector.id
+                    )
+                    detector_administrator_account = detector_administrator.get(
+                        "Administrator"
+                    )
+                    if detector_administrator_account:
+                        detector.administrator_account = (
+                            detector_administrator_account.get("AccountId")
+                        )
+                except Exception as error:
+                    logger.error(
+                        f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                    )
+                    continue
+
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def __list_members__(self, regional_clients):
+        logger.info("GuardDuty - listing members...")
+        try:
+            for detector in self.detectors:
+                try:
+                    regional_client = regional_clients[detector.region]
+                    list_members_paginator = regional_client.get_paginator(
+                        "list_members"
+                    )
+                    for page in list_members_paginator.paginate(
+                        DetectorId=detector.id,
+                    ):
+                        for member in page["Members"]:
+                            detector.member_accounts.append(member.get("AccountId"))
+                except Exception as error:
+                    logger.error(
+                        f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                    )
+                    continue
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
```

Comment:
At this level I think regional_client.region is not available because the function argument is `regional_clients` which is a list, please check it out.